spec/services/metadata_cloud_service_spec.rb

```diff
@@ -2,28 +2,31 @@
 require "rails_helper"
 require "webmock"
 
+WebMock.allow_net_connect!
+
 RSpec.describe MetadataCloudService, prep_metadata_sources: true do
   let(:mcs) { described_class.new }
-  let(:oid) { "16371253" }
+  let(:oid) { "16371272" }
```

Comment:
Same as above - change example back to `16371253` to shorten tests

Reply:
Fixed